content/docs/dvclive/ml-frameworks/pytorch-lightning.md

```diff
@@ -69,6 +82,52 @@ checkpointing at all as described in the
 
 ## Examples
 
+### Log model checkpoints
+
+Use `log_model` to save the checkpoints. DVCLive will first delete checkpoints
```

Comment:
It's still so complicated :( 

Sorry, Dave. I know you put tons of effort into this. And may be that's the best we can do ... but nonetheless we should recognize that's quire complex.

Let's start with this. Can it be just:

 > Use `log_model` to save the checkpoints (it will use `Live.log_artifact()` internally to save those). At the end of training, DVCLive will also mark the `best` the [`best_model_path`][`ModelCheckpoint`] (e.g. for then to be consumed in the Model Registry or automation scenarios).
 
 Even this one is still too complicated. Do ppl care about MR at this point? I don't know . Can it go after the code example?

Reply:
TBH I don't have that strong an opinion here at this point. I'm fine to go with that suggestion for now. We can always revisit.